src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneServiceIT.java

```diff
@@ -52,18 +58,18 @@ void cleanDatabaseTables() {
 
     @BeforeEach
     void setUp() {
-        subject = new MarkerGeneServiceImpl(markerGenesDao);
+        subject = new MarkerGeneService(markerGenesDao, cellTypeSearchDao);
     }
 
     @Test
     void getMarkerGeneProfileForTheValidExperimentAccession() {
-        assertThat(subject.getCellTypeMarkerGeneProfile("E-EHCA-2", "skin"))
+        assertThat(subject.getCellTypeMarkerGeneProfile("E-MTAB-5061", "http://purl.obolibrary.org/obo/UBERON_0001264"))
```

Comment:
May I ask why we use a http response rather than a static string here? The response is a html page.

Reply:
My initial impression was that this endpoint accepts the organism part and contacts Solr with the same organism part as input and gets cell types..etc

But After merging Alfonso Implemented Solr code, this got changed.

Now, this endpoint expectation is to accept `'Ontology'` terms instead of organism part as per @alfonsomunozpome. @alfonsomunozpomer  implemented Solr code like that to fetch cell types from Solr by Ontology terms as input. 
I Just followed this class: `CellTypeSearchDaoIT` to fix tests as I am calling Solr from the service layer. Hope you got some idea now :).